staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2627,7 +2627,7 @@ func (i *IngressDescriber) describeBackendV1beta1(ns string, backend *networking
 			}
 		}
 	}
-	return formatEndpointSlices(endpointSliceList.Items, sets.NewString(spName))
+	return formatEndpointSlices(endpointSliceList.Items, sets.New(spName))
```

Comment:
I think you meant to squash the last commit into the one before it?

Reply:
yes, squashed.